src/M6Web/Bundle/DaemonBundle/Command/DaemonCommand.php

```diff
@@ -209,20 +205,17 @@ public function run(InputInterface $input, OutputInterface $output)
         // Force the creation of the synopsis before the merge with the app definition
         $this->getSynopsis();
 
-        // Add the signal handler
-        if (function_exists('pcntl_signal')) {
```

Comment:
What is the point here with this removal ?

If is it to speed up the main loop, maybe we have to explicit it somewhere else or add a check on Bundle main file ?